app/assets/javascripts/backbone/views/loan_tab_view.coffee

```diff
@@ -51,3 +51,10 @@ class MS.Views.LoanTabView extends Backbone.View
             stepModal: @stepModal,
             locale: @locale
           )
+
+      when 'logs'
+        if MS.LogsListView
+          MS.LogsListView.refresh()
+        else
+          MS.LogsListView = new MS.Views.LogsListView(loanId: @loanId, el: '.loan-content #logs')
+          MS.LogsListView.refresh()
```

Comment:
Do I need to pass a done param here, if there is no done function for this instance of LogsListView?